src/components/DpMultiselect/DpMultiselect.vue

```diff
@@ -2,12 +2,22 @@
   <div>
     <vue-multiselect
       :close-on-select="closeOnSelect"
+      v-bind="conditionalProps"
+      :data-cy="dataCy"
```

Comment:
I wonder if it would enhance readability + is a recognized pattern to move other prop definitions to v-bind, too. Something like...

```suggestion
      v-bind="{
        ...conditionalProps,
        dataCy,
        deselectGroupLabel,
        deselectLabel,
        disabled,
        groupLabel,
        //...
      }"
```

However idk if this adds to code quality at all.